adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1576,14 +1543,12 @@ private boolean isAccessTokenReturned(final AuthenticationResult authResult) {
     }
 
     /**
-     * @param activity
-     * @param request
-     * @return false: if intent is not resolved or error in starting. true: if
-     *         intent is sent to start the activity.
+     * function returns false if intent is not resolved or error in starting.
```

Comment:
nit: This is not the standard way to javadoc. Looks like you're removing the param doc for private methods? If so, can we doc in this way, 1) for all the public and package visible class, do a standard javadoc 2) for the private method, like this case, do True if ..., false otherwise?